CoreProc/paymaya-laravel
# Create Artisan command to install webhooks

**What changed?**
- Added Artisan command to install webhooks (`paymaya:install-webhook`).
- Removed unused test.
- Removed unused command.
- Updated README.


**How can it be tested?**
- Install the package
- Configure credentials
- Configure webhooks
- Run `php artisan paymaya:install-webhook`
- Check registered webhooks.


**What are possible test scenarios?**
- Define webhook callback URLs as:
  - relative paths
  - route names
  - env variables
